executor/Cargo.toml

```diff
@@ -50,6 +51,7 @@ builtin_private_join_and_compute = []
 builtin_ordered_set_intersect = []
 builtin_rsa_sign = []
 builtin_face_detection = []
+builtin-principal-components-analysis = []
```

Comment:
Please change to `builtin_principal_components_analysis`, otherwise, this PR doesn't compile.